torch/onnx/_internal/fx/registration.py

```diff
@@ -147,9 +147,7 @@ def get_functions(
         internal_name_class = OpName.from_name_parts(
```

Comment:
```suggestion
        internal_name = OpName.from_name_parts(
```

it should be an instance rather than a class right?

Reply:
What do you think internal_name_instance? I tried to avoid internal_name. It seems like a string which is not.